dbt_subprojects/dex/macros/models/_project/in-flight-decoding/decoding_base.sql

```diff
@@ -0,0 +1,27 @@
+{% macro evm_event_decoding_base(
+    logs = null,
+    abi = null,
+    topic0 = null
+    )
+%}
+
+SELECT
+*
+
+FROM TABLE (
+    decode_evm_event (
+      abi => '{{abi}}',
+      input => TABLE (
+        SELECT l.* 
+        FROM {{logs}} l
+        WHERE topic0 = {{topic0}}
+            {% if is_incremental() %}
+            AND {{ incremental_predicate('block_time') }}
+            {% else %}
+            AND block_date >= (SELECT MIN(block_date) FROM {{logs}} WHERE topic0 = {{topic0}})
+            {% endif %}
+      )
+    )
+  )
```

Comment:
Updated `decode_evm_event` with incremental filters to try to fix CI incremental testing failure:
https://github.com/duneanalytics/spellbook/actions/runs/11125543803/job/30913453779?pr=6630#step:15:28

Reply:
The last incremental building error was produced by `left join` to duplicated pool decoded in the incremental run.

I think the problem may be solved in this update.

Reply:
Solved.

Reply:
nice catch!